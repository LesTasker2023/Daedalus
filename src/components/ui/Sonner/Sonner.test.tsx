import { describe, it, expect } from 'vitest';
import { render, screen } from '@testing-library/react';

describe('Sonner', () => {
  it('renders without crashing', () => {
    render(<div data-testid="sonner-test">Sonner Test</div>);
    expect(screen.getByTestId('sonner-test')).toBeInTheDocument();
  });

  describe('Snapshots', () => {
    it('matches basic snapshot', () => {
      const { container } = render(<div data-testid="sonner-container">Sonner Component Test</div>);
      expect(container.firstChild).toMatchSnapshot();
    });
  });

  it('renders test content correctly', () => {
    render(<div data-testid="content">Sonner Content</div>);
    expect(screen.getByTestId('content')).toHaveTextContent('Sonner Content');
  });
});
