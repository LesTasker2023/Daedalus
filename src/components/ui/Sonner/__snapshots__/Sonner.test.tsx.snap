// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Sonner > Snapshots > matches basic snapshot 1`] = `
<div
  data-testid="sonner-container"
>
  Sonner Component Test
</div>
`;
